dev/com.ibm.ws.webcontainer.servlet.3.1_fat.2/fat/src/com/ibm/ws/webcontainer/servlet31/fat/tests/UpgradeReadWriteTimeoutHttpUnit.java

```diff
@@ -10,6 +10,7 @@
 package com.ibm.ws.webcontainer.servlet31.fat.tests;
 
 import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
 
```

Comment:
Update copyright to include 2025?

Reply:
We discussed and decided to remove this change and investigate it outside of this PR. We don't want to introduce build breaks once we merge the netty code into integration.